src/main/cpp/t07_snake.cpp

```diff
@@ -17,11 +17,57 @@
 
 #include "t07_snake.h"
 #include <iostream>
-
+#include <iomanip>
 
 using namespace std;
 
 int t07_snake() {
-    
-    return 0;
+	int a[100][100];
+	int n;
+	int m;
+	cin >> n >> m;
+	int count = 0;
+	int numb = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'numb' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809)